tests/rule_based_profiler/domain_builder/test_domain_builder.py

```diff
@@ -168,19 +164,85 @@ def test_column_domain_builder_with_simple_semantic_type_included(
     domains: List[Domain] = domain_builder.get_domains(variables=variables)
 
     assert len(domains) == 2
+    # Assert Domain object equivalence.
     assert domains == [
         {
             "domain_type": "column",
             "domain_kwargs": {
                 "column": "event_type",
             },
-            "details": {"inferred_semantic_domain_type": "numeric"},
+            "details": {
+                "inferred_semantic_domain_type": "numeric",
+            },
         },
         {
             "domain_type": "column",
             "domain_kwargs": {
                 "column": "user_id",
             },
-            "details": {"inferred_semantic_domain_type": "numeric"},
+            "details": {
+                "inferred_semantic_domain_type": "numeric",
+            },
         },
     ]
+
+
+def test_multi_column_domain_builder(
+    alice_columnar_table_single_batch_context,
+    alice_columnar_table_single_batch,
+):
```

Comment:
I think Alice/Bobby/Bobster/Quentin should be moved to `integration` and we should write standard unit tests here.